pkg/ui/src/views/app/containers/layout/index.tsx

```diff
@@ -15,14 +14,16 @@ import AlertBanner from "src/views/app/containers/alertBanner";
  */
 export default class extends React.Component<RouterState, {}> {
   render() {
-    return <div>
-      <Helmet titleTemplate="%s | Cockroach Console" defaultTitle="Cockroach Console" />
-      <TimeWindowManager/>
-      <AlertBanner/>
-      <NavigationBar/>
-      <StickyContainer className="page">
-        { this.props.children }
-      </StickyContainer>
-    </div>;
+    return (
```

Comment:
+1 noticed this when adding the titles, meant to come back and do the reformat but forgot.  thanks!